### PR TITLE
Added CPU revision to result

### PR DESCRIPTION
**Pull request by joshhills**
This will allow Raspberry Pi users to determine their model.

### CODE REVIEW

File: lib/cpu.js [hunk 1]
```diff
@@ -83,17 +84,19 @@ function getCpu() {
         });
       }
       if (_linux) {
-        exec("cat /proc/cpuinfo | grep 'model name'", function (error, stdout) {
+        exec("cat /proc/cpuinfo | grep -E 'model name|Revision'", function (error, stdout) {
```
Comment (JoshHills) [inline lib/cpu.js]: The entire file could be parsed more effectively and returned. As it stands, I have a need for this specific feature and would like to continue using this library.

**Comment by sebhildebrandt** [comment]
Thank you Josh! I will review and merge it tomorrow ...

**Comment by JoshHills** [comment]
Hey I'll check this out and fix the merge conflicts when I get a chance, unless this feature is now redundant?

**Comment by sebhildebrandt** [comment]
Josh, sorry! I was super busy. Will try to merge this this tomorrow. I guess you do not have to do anything here ;-)

**Comment by sebhildebrandt** [comment]
Hi josh, me once again ...
As "Revision" seems not to be in a standard output of `/proc/cpuinfo`, could you please post the output of `cat /proc/cpuinfo` of your specific computer (just for one kernel)? Then I can make this more generic ;-)

**Comment by sebhildebrandt** [comment]
I updated cpu information by adding additional information like `vendor`, `family`, `model`, `stepping`, `cache`, `speedmin/max` but as I was not able to reproduce how to get `revision` information, I will close this for now. If possible, can you provide the upper requested information `cat /proc/cpuinfo` and `lscpu` of your specific installation, then I can see more clearly, what I have to add.

**Pull request closed without merging**